bin/prep-release.php

```diff
@@ -71,9 +71,6 @@ function get_files() {
         "Vagrantfile",
 
         "scripts/*/*.{sh}",
-        "scripts/*/ldap/*.{conf,ldif,py,sh}",
-        "scripts/centos/ldap/users",
-        "scripts/presets/*.{json}",
         "scripts/ssl/*.pem",
```

Comment:
Our old MO configs referred to these certs and our test files still read these. If these certs are no longer in sync with the MO configs, perhaps we should remove these and modify those tests to rely on certs from drivers-evergreen-tools.

I'll note that https://github.com/mongodb-labs/drivers-evergreen-tools/tree/master/.evergreen/orchestration/lib only contains a `client.pem` file. I'm not sure where the other files are.

Reply:
I'll take another look at those certs and report back.

Reply:
I've removed the certificates entirely. We need to test against the certificates from mongo-orchestration, so the path will now be read through the `SSL_DIR` environment variable. If that doesn't point to a valid path, SSL tests will be skipped. Users that test the driver against a server with SSL enabled need to set this environment variable to the path where the certificates rest. This is done automatically for evergreen but currently skipped on GitHub Actions (since we don't test SSL variants there anyways).